Corriendo el programa. ID de los resultados: 19022023173439
La corrida inicio a las: 19/02/2023 17:34!
 --- Ejecucion con 1 hilos
 --- Ejecucion con 2 hilos
 --- Ejecucion con 3 hilos
 --- Ejecucion con 4 hilos
La corrida termino a las: 19/02/2023 17:34!
python: can't open file 'Z:/juventino_velasquez/graficar.py': [Errno 2] No such file or directory
